### PR TITLE
Feat/create personal information section UI

### PR DESCRIPTION
**Pull request by AniaPiWo**

### CODE REVIEW

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

### app/(profile)/create-profile/page.module.scss
- The changes in this file look good.
- The new styles for `.main`, `.titleBox`, `.saveProfileBtn`, and `.formBox` are implemented correctly.
- The styles are consistent with the provided design.

### app/(profile)/create-profile/page.tsx
- The changes in this file look good.
- The `PersonalInfo` component is imported correctly.
- The `CreateProfilePage` component is updated to include the new UI elements.
- The styles from `page.module.scss` are applied correctly to the elements.

### src/assets/icons/ImportantIcon.tsx
- The new `ImportantIcon` component looks good.
- It renders an SVG icon with the correct dimensions and paths.
- The icon is filled with the appropriate color.

### src/components/PersonalInfo/PersonalInfo.jsx
- The new `PersonalInfo` component looks good.
- It renders a container with two sections: `.left` and `.right`.
- The text and input fields are implemented correctly.
- The `ImportantIcon` component is used correctly for the "Contact email" and "Bio" fields.

### src/components/PersonalInfo/PersonalInfo.module.scss
- The new styles for `.container`, `.left`, `.personalInfo`, `.right`, `.formTitle`, `.formInput`, `.formTextarea`, `.formItem`, and `.lettersCount` are implemented correctly.
- The styles are consistent with the provided design.

Overall, the code changes meet the requirements of creating the "Personal Information" section in the "Create a profile" page. The UI is implemented as close as possible to the provided design. The new components and styles are integrated correctly into the existing codebase.

Great job on implementing the changes! Let me know if you have any questions or need further assistance.

**Review by wawrzonnn — APPROVED** [review]

**Review by hansac91 — APPROVED** [review]

File: src/components/PersonalInfo/PersonalInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <div className={styles.left}>
+        <div>Personal Information</div>
+        <div className={styles.personalInfo}>
+          Share personal information to let the recruiters get to know you.{' '}
+        </div>
+      </div>
+      <div className={styles.right}>
+        <div className={styles.formItem}>
+          <div className={styles.formTitle}>Full name</div>
+          <input
```
Comment (Sumick) [inline src/components/PersonalInfo/PersonalInfo.jsx:18]: **nitpick:** Inputs should be separate components

File: app/(profile)/create-profile/page.tsx [hunk 2]
```diff
@@ -12,9 +13,16 @@ const CreateProfilePage = async () => {
   }
 
   return (
-    <div>
-      Create profile page
-      <br />
+    <div className={styles.main}>
+      <div className={styles.titleBox}>
+        <span>Create profile page</span>
+        <button className={styles.saveProfileBtn}>
```
Comment (Sumick) [inline app/(profile)/create-profile/page.tsx:19]: **issue:** Button should be separate component. Try to think of it as you would do Nerdux UI system

**Review by Sumick — CHANGES_REQUESTED** [review]
Buttons, and inputs should be separate components. Just like in Nerdux UI

File: src/components/CreateProfile/CreateProfileInput/CreateProfileInput.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+'use client'
+import React from 'react'
+import styles from './CreateProfileInput.module.scss'
+
+const CreateProfileInput = ({ placeholder }) => {
```
Comment (Sumick) [inline src/components/CreateProfile/CreateProfileInput/CreateProfileInput.jsx:5]: **nitpick:** This component should be more generic, like `TextInput` -  we are also missing required props here, like `onChange`, `value`. Exactly how you would do it in Nerdux

File: src/components/CreateProfile/CreateProfileTextArea/CreateProfileTextArea.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+'use client'
+import React from 'react'
+import styles from './CreateProfileTextarea.module.scss'
+
+const CreateProfileTextArea = ({ placeholder }) => {
```
Comment (Sumick) [inline src/components/CreateProfile/CreateProfileTextArea/CreateProfileTextArea.jsx:5]: **nitpick:** Same here - why `CreateProfileTextArea` if it's a generic component. We are also missing props interface here

**Review by Sumick — CHANGES_REQUESTED** [review]
Components are missing `interfaces`, and some props + input component should be more generic, just as in Nerdux. We should standardize it the same way, as in design system.

**Review by Sumick — APPROVED** [review]
Looks much better:) Good job!

**Pull request merged**